### PR TITLE
[RC2] Blacklist git_repo testing as discussed upstream

### PR DESCRIPTION
**Pull request by collivier**
https://github.com/kubernetes-sigs/kind/issues/2356

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 1]
```diff
@@ -255,6 +262,9 @@ skip:
   - [Feature:Volumes]
   - [Feature:Windows]
   - [NodeFeature:EphemeralStorage]
+  - PersistentVolumes.NFS
+  - should provision storage with different parameters
```
Comment (rihabbanday) [inline doc/ref_cert/RC2/chapters/chapter02.md:266]: Do we need to include the exclusions introduced above again under the "skip" section here?

Comment (gkunz) [reply doc/ref_cert/RC2/chapters/chapter02.md:266]: Same question here. This looks confusing to me.

Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter02.md:266]: @rihabbanday @gkunz I could remove them if you prefer. My point was to list the skip list as written in testcases.yml and to precise any regex which are not clear from RA2 sig tabs.

**Review by rihabbanday — APPROVED** [review]

**Comment by collivier** [comment]
Agreed in RA2 meeting

**Pull request merged**